README.md

```diff
@@ -4,3 +4,4 @@
 Dave Medrano
 
 ### Members
+Bonifacio, Andres
```

Comment:
@evadonardem , kindly follow format
- <LASTNAME>, <Firstname>

Reply:
I am done with the requested changes.